### PR TITLE
Design changes for release

### PR DESCRIPTION
**Pull request by CarolineLaw**
see attached ticket

### CODE REVIEW

File: app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt [hunk 1]
```diff
@@ -250,11 +270,15 @@ class EditKeyboardFragment : BaseFragment() {
...
             it.action = {
-                parentFragmentManager
-                    .beginTransaction()
-                    .replace(R.id.settings_fragment_container, EditPresetsFragment())
-                    .addToBackStack(null)
-                    .commit()
+                if (isCategory) {
+                    parentFragmentManager.popBackStack()
+                } else {
+                    parentFragmentManager
```
Comment (nicholas-cook) [inline app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt:276]: Should this one also be `parentFragmentManager.popBackStack()`?

File: app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt [hunk 2]
```diff
@@ -124,14 +126,11 @@ class EditKeyboardFragment : BaseFragment() {
         binding?.backButton?.action = {
             val isEditing = arguments?.getBoolean(KEY_IS_EDITING) ?: false
             val textChanged = binding?.keyboardInput?.text.toString() != phrase?.getLocalizedText()
-            if (isCategory) {
+            val categoryTextChanged = binding?.keyboardInput?.text.toString() != category?.getLocalizedText()
```
Comment (nicholas-cook) [inline app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt:129]: Are you changing `keyboard_input` to an `AppCompatTextView` here or in another PR?

Comment (CarolineLaw) [reply app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt:129]: another PR :]

File: app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt (hunk 1 above)
Comment (CarolineLaw) [reply app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt:276]: yes i think so, thank you!

File: app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt [hunk 3]
```diff
@@ -273,11 +273,7 @@ class EditKeyboardFragment : BaseFragment() {
                 if (isCategory) {
                     parentFragmentManager.popBackStack()
                 } else {
-                    parentFragmentManager
-                        .beginTransaction()
-                        .replace(R.id.settings_fragment_container, EditPresetsFragment())
-                        .addToBackStack(null)
-                        .commit()
+                    parentFragmentManager.popBackStack()
```
Comment (nicholas-cook) [inline app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt:276]: You should be able to get rid of the if-statement now if they're both doing the same thing

**Pull request merged**